### PR TITLE
Added default decoding

### PR DESCRIPTION
**Pull request by ShahafBenYakir**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/37721

## Description
Added a fallback mechanism that works with utf-8 to handle unsuccessful decoding process made with auto-detected encoding.

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/EWS/Integrations/EWSO365/EWSO365.py [hunk 1]
```diff
@@ -2066,9 +2071,23 @@ def parse_incident_from_item(item):
...
+                        try:
+                            data = attached_email_bytes.decode(default_encoding)
+                        except Exception as e:
+                            demisto.debug(f'attached_email bytes:\n{attached_email}')
+                            demisto.debug(e)
+
                     file_result = fileResult(
                         get_attachment_name(attachment.name) + ".eml",
-                        attached_email_bytes.decode(encoding),
+                        data,
```
Comment (yaakovi) [inline Packs/EWS/Integrations/EWSO365/EWSO365.py:2090]: you might get a `used before assigned` error in case both encoding options fails

File: Packs/Gmail/Integrations/Gmail/Gmail.yml [hunk 2]
```diff
@@ -2242,7 +2242,7 @@ script:
       description: A name that appears in the "From:" header for email sent using
         this alias.
       type: String
-  dockerimage: demisto/google-api:1.0.0.18525
+  dockerimage: demisto/google-api:1.0.0.20520
```
Comment (Itay4) [inline Packs/Gmail/Integrations/Gmail/Gmail.yml:2245]: why did you change this?

**Review by Itay4 — APPROVED** [review]
